api/provider/aws/service_read_test.go

```diff
@@ -85,4 +85,5 @@ func TestService_makeServiceModel(t *testing.T) {
 	assert.Equal(t, "lb_name", result.LoadBalancerName)
 	assert.Equal(t, "svc_id", result.ServiceID)
 	assert.Equal(t, "svc_name", result.ServiceName)
+	assert.Equal(t, "stateless", result.ServiceType)
```

Comment:
I'm not a fan of `ServiceType` and `TaskType`; shouldn't it be `DeployCompatibility` ?

Reply:
Replacing with `Stateful` boolean field as per https://github.com/quintilesims/layer0/pull/567#discussion_r171947782 ✅